index.html

```diff
@@ -19,7 +20,13 @@ <h5>9:41</h5>
     </div>
     <div class="lower-nav">
       <h3>Tarek Yosry</h3>
-      <img src="./images/Union.png" alt="hamburger menu">
+      <img class="hamburger-menu" src="./images/Union.png" alt="hamburger menu">
+      <ul class="pop-up">
+        <li><a href="">Portfolio</a></li>
+        <li><a href="">About</a></li>
+        <li><a href="">Contact</a></li>
+      </ul>
+      <i class="bi bi-x-lg close-icon"></i>
```

Comment:
- [ ] The close(X) button is a little bigger than available on the template. Please decrease the size of the close(X) button and make it matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A838) design.